src/Kdyby/Redis/RedisSessionHandler.php

```diff
@@ -228,7 +236,8 @@ protected function lock($id)
 	 */
 	private function formatKey($id)
 	{
-		return self::NS_NETTE . $id;
+		$key = $this->keyPrefix . $id;
```

Comment:
it makes more sense to have the instance prefix at the beggining of the key to me